Add linux/windows keybindings

sylvaindumont opened the pull request:
Add alt-shift-t keybinding to open terminal.
It allows non mac user to open terminal with keybinding.

It could help with #6 if we remove cmd-shift-t.

daviwil commented:
+1 for platform-agnostic keybindings

jsmecham commented:
Agree that `cmd-shift-t` is probably not a good default, although that is what I would choose to map personally. We shouldn't clobber existing conventions out of the box, though.

daviwil commented:
I noticed that they keybinding-resolver package uses [these CSS classes](https://github.com/atom/keybinding-resolver/blob/master/keymaps/keybinding-resolver.cson) to do platform-specific bindings:

```
{
  ".platform-darwin": {
    "cmd-.": "key-binding-resolver:toggle"
  },
  ".platform-win32": {
    "ctrl-.": "key-binding-resolver:toggle"
  },
  ".platform-linux": {
    "ctrl-.": "key-binding-resolver:toggle"
  }
}
```

I wonder if we could do something similar?

sylvaindumont commented:
as proposed by @daviwil I added platform specific bindings, it doesn't change anything for mac users, `cmd-shift-t` still open terminal instead of reopening last closed tab.

`alt-shift-t` open terminal for windows/linux users to avoid introducing #6 on windows/linux

jsmecham committed:
Removed Cmd-K as a keybinding for clearing the terminal.

I hope to be able to restore this, but right now it conflicts with a whole slew of keybindings in Atom on the Mac.

jsmecham commented:
I'm going to address this in #66. Would you mind rebasing this PR on that branch? I'll merge it there and probably make a bunch of changes for Windows and Mac, as well.

jsmecham commented on keymaps/terminal.json:
```diff
@@ -1,10 +1,26 @@
+    "alt-shift-t": "terminal:open"
+  },
+  ".platform-win32 terminal-view": {
+    "ctrl-c": "terminal:copy",
```
I don't think we want to bind to Ctrl-C on Windows, as that's a standard shortcut for interrupting a program when running in a terminal.

sylvaindumont replied:
On linux too. I wanted to stay close of mac keybinding until we find better ideas, will replace by #62 proposition `ctrl` `shift` `c`/`v`/`k`
`cmd` `shift` `k` on mac could solve what you fix with 713fe60d74b3b96b20c40aab04e57352d03a886e

sylvaindumont committed:
platform specific bindings

sylvaindumont committed:
better copy/paste keybindings

sylvaindumont commented:
@jsmecham rebased on #66 and copy/paste conflicts fixed

forresthopkinsa commented:
What's the status of this?
